config/application.rb

```diff
@@ -9,7 +9,7 @@
 require "active_record/railtie"
 # require 'active_storage/engine'
 require "action_controller/railtie"
-# require 'action_mailer/railtie'
+require "action_mailer/railtie"
```

Comment:
Even when we remove all of the email stuff from devise, it _still_ gets angry if we don't have action mailer.  Will have to look more at this later.

Reply:
I'm curious to know what was happening. I removed this and was able to run full specs, server and console. is there some sort of deployment/bundling thing?

Reply:
never mind. I saw [this](https://github.com/go-between/musicbox-api/pull/61/commits/f388fc99d26dc6c02803c5500cb8e4a6d00985d3).